src/pages/UpdatePassword/index.js

```diff
@@ -0,0 +1,91 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import FooterBrand from 'Components/compounds/FooterBrand';
+import BasicNav from 'Components/compounds/BasicNav';
+import InputField from 'Components/atoms/InputField/InputField';
+import { resetVerifiedUserPassword } from 'Actions/resetPasswordActions';
+import validatePassword from 'Utils/validatePassword';
+import Alert from 'Utils/helpers/alert';
+import logo from 'Images/logo.png';
+import './style.scss';
+
+export class UpdatePassword extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      userInfo: {
+        password: '',
+        confirmPassword: ''
+      }
+    };
+  }
+
+  handleChange = (e) => {
+    const { userInfo } = this.state;
+    const newUserInfo = { ...userInfo };
+    newUserInfo[`${e.target.name}`] = e.target.value;
+    this.setState({
+      userInfo: newUserInfo
+    });
+  }
+
+  handleSubmit = (e) => {
+    e.preventDefault();
+    const { userInfo } = this.state;
+    // eslint-disable-next-line
```

Comment:
👀

Reply:
Beht why?

Reply:
`[eslint] 'resetVerifiedUserPassword' is already declared in the upper scope.`
The above is the **eslint** flag that I disabled. 

It flags because it is taking the `resetVerifiedUserPassword` redux-action function imported into the file and the one available to the component through `props` to be the same.